dapp/src/utils/anonymousVoting.ts

```diff
@@ -165,6 +165,10 @@ export async function computeAnonymousVotingData(
       /* ignore – default 1 */
     }
 
+    if (voteChoiceIdx === 2 && maxWeight === 1) {
+      maxWeight = 50000;
```

Comment:
This is not good. This should be using the badge.verified so that if we change the value, this would change automatically.